test/blackbox-tests/test-cases/odoc/warnings.t/run.t

```diff
@@ -25,9 +25,9 @@ These packages are in a nested env, the option is disabled, should success with
 In release mode, no error:
 
   $ dune build -p foo_doc,foo_lib @doc
-          odoc _doc/_odoc/pkg/foo_doc/page-foo.odoc
+  (cd _build/default/_doc/_odoc/pkg/foo_doc && /home/dim/.opam/4.12.0/bin/odoc compile --pkg foo_doc -o page-foo.odoc ../../../../foo_doc/foo.mld)
```

Comment:
Oops, I missed that this error is non-deterministic (see `dim` in the path), so this breaks the tests.

Reply:
Yeah, just came across it. Fixing it now